nmostesting/IS05Utils.py

```diff
@@ -40,6 +40,10 @@ def get_valid_transports(self, api_version):
         if self.compare_api_version(api_version, "v1.1") >= 0:
             valid_transports.append("urn:x-nmos:transport:websocket")
             valid_transports.append("urn:x-nmos:transport:mqtt")
+
+        if not with_transport_file_support:
+            valid_transports.remove("urn:x-nmos:transport:websocket")
+            valid_transports.remove("urn:x-nmos:transport:mqtt")
```

Comment:
I'm a little confused. My understanding was that `get_valid_transports` , `with_transport_file_support=True` would return only `rtp` and `rtp.mcast`. Is that right? As I think that this will return `rtp`, `rtp.mcast` as well as`websocket` and `mqtt` when `with_transport_file_support=True`.

But either way could the conditions be combined into a single statement I think:
```suggestion
        if self.compare_api_version(api_version, "v1.1") >= 0 and not with_transport_file_support:
            valid_transports.append("urn:x-nmos:transport:websocket")
            valid_transports.append("urn:x-nmos:transport:mqtt")
```

Reply:
How about:

```python
    def get_valid_transports(self, api_version, include_transports_without_transport_file=True):
        """Identify the valid transport types for a given version of IS-05"""
        valid_transports = ["urn:x-nmos:transport:rtp",
                            "urn:x-nmos:transport:rtp.mcast",
                            "urn:x-nmos:transport:rtp.ucast",
                            "urn:x-nmos:transport:dash"]
        if self.compare_api_version(api_version, "v1.1") >= 0 and include_transports_without_transport_file:
            valid_transports.append("urn:x-nmos:transport:websocket")
            valid_transports.append("urn:x-nmos:transport:mqtt")
        return valid_transports
```

Reply:
Yep, that's better 👍